### ISSUE TITLE
Catalog Error: Scalar Function with name st_points does not exist!

### ISSUE THREAD

**Issue by jeremysquires-sensorup** [duckdb/duckdb_spatial#354]
Steps to reproduce:

INSTALL spatial;
LOAD spatial;

select st_points('LINESTRING(1 1, 2 2)'::geometry);

Catalog Error: Scalar Function with name st_points does not exist!
Did you mean "ST_NPoints"?
LINE 1: select st_points('LINESTRING(1 1, 2 2)'::geome...

duckdb 1.0.0 on MacOS

**Comment by Maxxen** [comment]
Hello! Thanks for reporting this issue
The ST_Points function was implemented after DuckDB 1.0.0 was released, and since spatial follows the same release calendar as the main duckdb project this function is only available on the nightly builds. You can install the latest nightly build (built against the `main` branch on this repository) for v1.0.0 by executing:

```
FORCE INSTALL spatial FROM 'http://nightly-extensions.duckdb.org' ;
```
And then restarting duckdb.

**Comment by jeremysquires-sensorup** [comment]
Thanks for the quick response!
